### PR TITLE
Fetching user location weather

### LINKED ISSUE #13: [JS] Add current user city weather to the page on load
When the user opens the page, a prompt box will appear that asks to allow the API to get its location, after that, Current user city weather information will appear on the page.

### PR DESCRIPTION
**Pull request by AseelL**
Resolves #13

### CODE REVIEW

File: js/home.js [hunk 1]
```diff
@@ -8,6 +8,35 @@ let weekInfoContainer = document.createElement("div");
...
+  }
+ function showError(error) {
+
+      if (error.PERMISSION_DENIED) {
+        alert("Please allow access to your location to show your weather data");
+      }
+    }
+}
+
+
```
Comment (zaher-aa) [inline js/home.js]: Format your code.

File: js/home.js [hunk 2]
```diff
@@ -27,8 +56,9 @@ function fetchWeatherData(searchTerm) {
     dailyData,
     apiKey = "d0139168498c4f66d3fb31b4d374f145";
   fetch(
-    `https://api.openweathermap.org/data/2.5/weather?q=${searchTerm}&appid=${apiKey}`
-  )
+
+      `https://api.openweathermap.org/data/2.5/weather?q=${searchTerm}&appid=${apiKey}`
+    )
```
Comment (zaher-aa) [inline js/home.js]: format this too

**Review by zaher-aa — CHANGES_REQUESTED** [review]

File: js/home.js (hunk 2 above)
Comment (AseelL) [reply js/home.js]: done

**Review by sara219 — CHANGES_REQUESTED** [review]
write a description comment before the functions, describe what the function do

**Review by zaher-aa — DISMISSED** [review]

**Review by sara219 — DISMISSED** [review]

**Review by zaher-aa — APPROVED** [review]

**Review by sara219 — APPROVED** [review]

**Pull request merged**